.github/workflows/ci.yml

```diff
@@ -289,6 +289,56 @@ jobs:
     - name: Interface test
       run: cmake --build build --target test_cmake_build
 
+  centos-nvhpc:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        centos:
+          - 7  # GCC 4.8
+          - 8
```

Comment:
Any reason you are building twice, with identical compilers, just with a different base system? You aren't using the host GCC.

Reply:
Hi @henryiii ,

There are differences, e.g. Python and cmake. But in general, I would consider CI to be an "environment test" rather than a test of some specific compiler version.  The later has little practical value, as for me. 

Best regards,
Andrii